docs/en/latest/deployment-modes.md

```diff
@@ -155,15 +155,273 @@ deployment:
 
 ## Standalone
 
-In the standalone deployment mode, APISIX is deployed as a data plane and it reads in configurations from a YAML file (`apisix.yaml`) in the local file system.
+Turning on the APISIX node in Stand-alone mode will no longer use the default etcd as the configuration center.
 
-This deployment mode is useful when you have to declaratively define the configuration or when you are using a different configuration center other than etcd.
+This method is more suitable for two types of users:
 
-To configure APISIX in standalone mode:
+1. kubernetes(k8s)：Declarative API that dynamically updates the routing rules with a full yaml configuration.
+2. Different configuration centers: There are many implementations of the configuration center, such as Consul, etc., using the full yaml file for intermediate conversion.
 
-```yaml title="conf/config.yaml"
+The routing rules in the `conf/apisix.yaml` file are loaded into memory immediately after the APISIX node service starts. And every time interval (default 1 second), will try to detect whether the file content is updated, if there is an update, reload the rule.
+
+*Note*: When reloading and updating routing rules, they are all hot memory updates, and there will be no replacement of working processes, it is a hot update.
```

Comment:
_Note_: Reloading and updating routing rules are all hot memory updates. There is no replacement of working processes, it's a hot update.